### DISCUSSION TITLE
Floating point exception (core dumped)

### DISCUSSION THREAD

**Discussion by evilblade666** [Comfy-Org/ComfyUI#5274] [Q&A]
Hey folks!
I've installed ComfyUI, Nvidia's pips and requirements. ComfyUI start, I can access it through my browser. Then when I try the generation - it loads the model (at least that's what I get from seeing the console output), but then when it comes to the actual generation (I assume) I get this "floating point exception (core dumped)" error and it crashes.
Can someone tell me what can I do? For starters - where can I look for some more detailed logs to see perhaps which exactly component fails?..

**Comment by JeB-0** [comment]
If you are on a linux system, check the log for errors similar to the following:

For Debian 12, `$sudo journalctl | grep libtorch_cpu.so` in a terminal.

> Dec 18 15:45:37 mesher kernel: traps: python[4315] trap divide error ip:7f14af75e783 sp:7f139f076bd0 error:0 in libtorch_cpu.so[7f14a23b6000+1037f000]
> Dec 18 18:16:17 mesher kernel: traps: pt_main_thread[7512] trap divide error ip:7f91e5bf8003 sp:7f90d3db2b80 error:0 in libtorch_cpu.so[7f91d5a3d000+1317f000]

The two errors are from different PyTorch versions. 

Those log entries are from a debian 12 system that I run LLMs and 3d mesh calculations on. It has an old motherboard and a couple of semi-decent GPUs because almost everything it does is CUDA. The glaring difference between it and another Debian 12 system that will run ComfyUI is the older CPU only supports AVX1. I think the newer PyTorch still intends to work without AVX2, but it is likely the culprit.  I would like to know what CPU you are having trouble running it on to see if I am barking up the right tree. 

EDIT: my older one doesn't support AVX at all. That is almost certainly the problem for mine. I'll still give a check to see if modern PyTorch can be compiled without AVX.

I'll make a brief run at this the week after Christmas, but I'll probably wind up throwing a slightly less old CPU at the problem.

**Reply by JeB-0** [reply]
Update for anyone else that finds this. Both the machine that can run ComfyUI and the one with the divide error are running nvidia-cublas-cu11==11.11.3.6 from the Debian package manager. Having already fruitlessly exhausted minutes of effort to pin down the error, I've given up and pulled a less ancient motherboard out of the scrap bin to get mine up and running on both machines.

**Comment by CHNtentes** [comment]
I solved this issue on H20, not sure if it works for your card.

You need to check your conda list | grep cublas first. Ensure there is only one cublas and that the version is greater than 12.3, then run pip install nvidia-cublas-cu12==12.4.5.8.

**Reply by tsy77** [reply]
do you have a docker image?
